digest/checksums.py

```diff
@@ -64,7 +64,7 @@ def compute_tarsum(fp, json_data):
             hashes.append(h)
         hashes.sort()
     except tarfile.ReadError as e:
-        if e.message != "empty file":
+        if str(e) != "empty file":
```

Comment:
This may not work; we need to test this. Please add a TODO

Reply:
Added TODO to add an empty tarfile test.